book/src/specs/kimchi.md

```diff
@@ -1148,10 +1148,10 @@ To create the index, follow these steps:
 
    To do this, for each table:
 
-	- Update the corresponding entries in a table id vector (of size the domain as well)
+~ - Update the corresponding entries in a table id vector (of size the domain as well)
```

Comment:
You need to update your `cargo spec`

Reply:
Oh thanks!! So that was it, CI was failing and I was not sure what was happening with the specs 👍🏻